fastlane/Fastfile

```diff
@@ -160,7 +162,7 @@ import 'lanes/release_management_in_ci.rb'
 
 default_platform(:android)
 
-before_all do |lane|
+before_all do |_lane|
```

Comment:
I believe RuboCop prefixed this with `_` because it's unused. But if that's the case, we might as well remove it, no?

```suggestion
before_all do
```

Reply:
Yes, this was auto-fixed by RuboCop.
In principle I agree, but at the same time, the signature does help to document that this is a special Fastlane thing 🤔

Reply:
My two cents are that it's up to us devs to look into the documentation to discover what optional parameters the method we use have. I prefer the code to be leaner, every unused parameter begs the question "why is this here?"

But, not something I feel so strongly about to delay this PR for it.

Reply:
True, the `_` RuboCop requests brings in a bit more obscurity to the fact parameter is there. Decided to go with your suggestion: cc10f9df0b89255fcf721a723d5c90359f32a325